develop/interoperability/test-and-debug.md

```diff
@@ -122,4 +101,6 @@ One of the key additions is the [`decl_test_bridges`](https://github.com/parityt
     }
     ```
 
-Utilizing the capabilities of the xcm-emulator, developers can effectively design, test, and optimize cross-chain functionality, fostering interoperability within the Polkadot ecosystem.
\ No newline at end of file
+By leveraging these macros, developers can customize their testing networks by defining relay chains and parachains tailored to their needs. For guidance on implementing a mock runtime for a Polkadot SDK-based chain, refer to the [Pallet Testing](/develop/parachains/testing/pallet-testing/){target=\_blank} article. 
+
+This framework enables thorough testing of runtime and cross-chain interactions, enabling developers to effectively design, test, and optimize cross-chain functionality.
```

Comment:
I'd probably add an example of a test with this tool.
Also, offer some criteria to choose one over the other:
- Emulator if
  - No live runtime yet
  - Want to tweak a lot of configurations (emulated chains are different from live chains)
  - Want rust tests
- Otherwise chopsticks, lets you use real live state